content/src/view.js

```diff
@@ -245,16 +250,21 @@ function initialPaneState() {
 }
 
 function setOnCallback(tag, cb) {
-  state.callbacks[tag] = cb;
+  if (state.callbacks[tag] == null){
+    state.callbacks[tag] = [];
+  }
+  state.callbacks[tag].push(cb);
 }
 
 function fireEvent(tag, args) {
   if (tag in state.callbacks) {
-    var cb = state.callbacks[tag];
-    if (cb) {
-      cb.apply(null, args);
-    }
-  }
+    var cbs = state.callbacks[tag].slice();
```

Comment:
Yeah, there is a reason for the slice().  We should add a comment explaining it....

The reason is: if you add (or someday, remove) an event handler while event propagation is happening, it can mess up the array.  So copying the array before looping and calling callbacks a standard thing to do.